lib/signer.ex

```diff
@@ -98,20 +104,20 @@ defmodule AntikytheraAws.Signer do
       [String.slice(amz_date, 0..7), region, service, @termination_string] |> Enum.join("/")
     end
 
-    defunp canonical_request(method       :: v[Method.t],
-                             path         :: v[UPath.t],
-                             payload      :: v[String.t],
-                             cheaders_str :: v[String.t],
-                             sheaders_str :: v[String.t],
-                             params       :: [{String.t, String.t}]) :: String.t do
+    defunp canonical_request(method         :: v[Method.t],
+                             path           :: v[UPath.t],
+                             payload_sha256 :: v[binary],
```

Comment:
After encoding SHA256 into hex it's actually a `String.t`.

Reply:
Oh, I didn't know that.
fixed in 4586ca8